sources/rest_api_pipeline.py

```diff
@@ -28,19 +24,31 @@ def load_github() -> None:
             },
         },
         "resources": [
-            # "pulls", <- This is both name and endpoint path
+            # This is a simple resource definition,
+            # that uses the endpoint path as a resource name:
+            # "pulls",
+
+            # Alternatively, you can define the endpoint as a dictionary
             # {
-            #     "name": "pulls",
+            #     "name": "pulls", # <- Name of the resource
             #     "endpoint": "pulls",  # <- This is the endpoint path
             # }
+
+            # Or use a more detailed configuration:
             {
                 "name": "issues",
                 "endpoint": {
                     "path": "issues",
+                    # Query parameters for the endpoint
                     "params": {
                         "sort": "updated",
                         "direction": "desc",
                         "state": "open",
+                        # Define `since` as a special parameter
```

Comment:
Maybe add a not that the name can be any valid name not only `since` or is it not the case?

Reply:
It'a s `since` query string parameter from the API, probably need to clarify that